wasted-app/RestaurantListControl.cs

```diff
@@ -40,18 +40,11 @@ public void AddAllRestaurants()
             {
                 AddRestaurnt(r);
             }
-
-
-            for (int i = 1; i <= 10; i++)
-            {
-                restaurantPanel.Controls.Add(new RestaurantControl(i.ToString()));
-            }
-            
         }
 
         private void AddRestaurnt(Restaurant restaurant)
```

Comment:
still there are some typos in file: "Restaurnt"

Reply:
Fixed